cityofaustin/atd-moped
# Update funding table activity log entries

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/11091

## Testing
**URL to test:** 
https://deploy-preview-921--atd-moped-main.netlify.app/moped/projects/1900?tab=activity_log


**Steps to test:**

Add a new funding source. 
Edit existing funding source. one field updated, multiple updated. 
Delete funding source. 

go to activity log tab, you may have to refresh to see the updates. See that your changes are reflected in some way. 

![Screen Shot 2023-01-12 at 4 35 55 PM](https://user-images.githubusercontent.com/12474808/212196066-b6458105-ee67-4a3f-ab21-267d4ff9ef32.png)


---
#### Ship list
